birdhouse/components/weaver/default.env

```diff
@@ -96,7 +96,13 @@ export WEAVER_WPS_PROVIDERS_RETRY_COUNT=5
 # control interval time between retries (duration in seconds, counts toward maximum timeout)
 export WEAVER_WPS_PROVIDERS_RETRY_AFTER=5
 
+export WEAVER_MONGODB_DATA_DIR='${DATA_PERSIST_ROOT}/mongodb_weaver_persist'
+
+export WEAVER_WPS_PRIVATE_DIR='${DATA_PERSIST_ROOT}/wps_private'
```

Comment:
This one might need to be revisited. 
Might be deprecated considering the switch to `wpsoutputs` with nested user-context managed by Cowbird.

Reply:
@fmigneault are you suggesting that this shouldn't be included in this PR or just that this might be deprecated some time in the future?

Reply:
Yes. I think it will be deprecated eventually.